Cargo.toml

```diff
@@ -15,6 +15,10 @@ anyhow = "1.0"
 env_logger = "0.9.0"
 lazy_static = "1.4.0"
 canonical-path = "2.0.2"
-
+image = "0.24.2"
+pdf_render = { git = "https://github.com/pdf-rs/pdf_render" }
+pdf = { git = "https://github.com/pdf-rs/pdf", features = ['threads'] }
+pathfinder_rasterize = { git = "https://github.com/nagato-chan/pathfinder_rasterize" }
```

Comment:
We need changes from this fork either be merged into original repo or maintained in an [ARK Builders organization](https://github.com/ARK-Builders/).

Reply:
Better in our organization actually, because the original repo seems to be not really maintained.